src/components/DatasetEditor/EditableTitleText.js

```diff
@@ -10,53 +10,18 @@ const useStyles = makeStyles({
   },
 })
 
-const SAVE_WAIT = 2000
-
 export default ({ value, onChange }) => {
```

Comment:
nice, didn't know if you'd be able to do this and keep it controlled but looks simple

Reply:
yes, I added story to check the changed values for the parent component and it's working fine for controlled component.

you can double check as well